jenkins/pipelines/ci/tidb/tidb_ghpr_build.groovy

```diff
@@ -209,64 +207,34 @@ try {
-                                    if (user_bazel(ghprbTargetBranch) != "")  {
-                                        sh """
-                                        if make bazel_build; then
-                                            touch importer.done
-                                            touch tidb-server-check.done
-                                        else 
-                                            touch importer.fail
-                                            touch tidb-server-check.fail
-                                            exit 1
-                                        fi
-                                        """
-                                    } else {
-                                        sh """
-                                        nohup bash -c "if make importer ;then touch importer.done;else touch importer.fail; fi"  > importer.log &
-                                        nohup bash -c "if WITH_CHECK=1 make TARGET=bin/tidb-server-check ;then touch tidb-server-check.done;else touch tidb-server-check.fail; fi" > tidb-server-check.log &
-                                        make
-                                        """
-                                    }
-                                }else{
-                                    if (user_bazel(ghprbTargetBranch) != "")  {
-                                        sh """
-                                        if make bazel_build; then
-                                            touch importer.done
-                                            touch tidb-server-check.done
-                                        else 
-                                            touch importer.fail
-                                            touch tidb-server-check.fail
-                                        fi
-                                        touch tidb-server-check.done
-                                        """
-                                    } else {
-                                        sh """
-                                        nohup bash -c "if make importer ;then touch importer.done;else touch importer.fail; fi"  > importer.log &
-                                        nohup bash -c "if  WITH_CHECK=1 make TARGET=bin/tidb-server-check ;then touch tidb-server-check.done;else touch tidb-server-check.fail; fi" > tidb-server-check.log &                                    
-                                        make
+                                if (user_bazel(ghprbTargetBranch) != "")  {
+                                    sh """
+                                    if make bazel_build; then
+                                        touch importer.done
                                         touch tidb-server-check.done
-                                        """
-                                    }
+                                    else 
+                                        touch importer.fail
```

Comment:
`touch xxx` is redundant commands, how about remove those lines.

Reply:
I will deal in new pipelines, won't adjust in old items.